_includes/assets/sass/components/_button.scss

```diff
@@ -0,0 +1,23 @@
+.button {
+  --primary-color: #{$tbds-brand-red};
+  --background-color: white;
+
+  background-color: var(--primary-color);
+  border-radius: 100rem;
+  box-shadow: 0 0 0 2px var(--background-color), 0 0 0 5px var(--background-color);
```

Comment:
I'm probably missing something but is this box-shadow necessary? The button seems to behave the same without it because of its hover state.

Reply:
Oh! Good spot!